content/desktop/contributing-and-collaborating-using-github-desktop/committing-and-reviewing-changes-to-your-project.md

```diff
@@ -35,48 +35,51 @@ As you make changes to files in your text editor and save them locally, you will
+
+{% endnote %}
 
-![Unselected lines in a file](/assets/images/help/desktop/partial-commit.png)
+1. In the top right corner, use {% octicon "gear" aria-label="The Gear icon" %} to select **Unified** or **Split** to review your changes.
+  ![Gear icon with unified and split diffs](/assets/images/help/desktop/gear-diff-select.png)
+2. The lines that will be included in the commit are highlighted in blue. To exclude the change, click one or more changed lines so the blue disappears.
+  ![Unselected lines in a file](/assets/images/help/desktop/partial-commit.png)
 
-#### Discarding changes
+### 3. Discarding changes
+If you have uncommitted changes that you don't want, you can remove the changes by discarding them. You can discard all uncommitted changes in one file, a range of files, or discard all changes in all files since the last commit.
 
-You can discard all the uncommitted changes in one file, a range of files, or discard all changes in all files since the last commit.
+Discarded changes are saved in a dated file in the Trash and you can recover them until the Trash is emptied.
 
-{% mac %}
+#### Discarding changes in one or more files
 
 {% data reusables.desktop.select-discard-files %}
 {% data reusables.desktop.click-discard-files %}
   ![Discard Changes option in context menu](/assets/images/help/desktop/discard-changes-mac.png)
 {% data reusables.desktop.confirm-discard-files %}
   ![Discard Changes button in the confirmation dialog](/assets/images/help/desktop/discard-changes-confirm-mac.png)
 
-{% tip %}
+#### Discarding changes in one or more lines
+You can discard one or more added lines in a single file before you commit them.
 
-**Tip:** The changes you discarded are saved in a dated file in the Trash and you can recover them until the Trash is emptied.
-
-{% endtip %}
+{% note %}
 
-{% endmac %}
+**Note:** Discarding single lines is disabled in a group of changes that adds and removes lines.
 
-{% windows %}
+{% endnote %}
 
-{% data reusables.desktop.select-discard-files %}{% data reusables.desktop.click-discard-files %}
-  ![Discard Changes option in context menu](/assets/images/help/desktop/discard-changes-win.png)
-{% data reusables.desktop.confirm-discard-files %}
-  ![Discard Changes button in the confirmation dialog](/assets/images/help/desktop/discard-changes-confirm-win.png)
+In the list of changed files, right click on the line you want to discard and select **Discard added line**.
 
-{% tip %}
+  ![Discard single line in the confirmation dialog](/assets/images/help/desktop/discard-single-line.png)
 
-**Tip:** The changes you discarded are saved in a file in the Recycle Bin and you can recover them until it is emptied.
+In the list of changed files, right click the group of lines you want to discard and select **Discard added lines**. 
 
-{% endtip %}
+  ![Discard multiple lines in the confirmation dialog](/assets/images/help/desktop/discard-multiple-lines.png)
```

Comment:
I think it's misleading to call it "multiple added lines" because that could imply removing lines that are far apart, like L5 and L100, (as you can with files)... when really they have to be grouped together.

```suggestion
  ![Discard a group of added lines in the confirmation dialog](/assets/images/help/desktop/discard-multiple-lines.png)
```